enginetest/queries/json_table_queries.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package queries
+
+import (
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+var JSONTableQueryTests = []QueryTest{
+	{
+		Query:    "SELECT * FROM JSON_TABLE(NULL,\"$[*]\" COLUMNS(x int path \"$.a\")) as t;",
```

Comment:
For all these tests, use `'` instead of `"` for your quote delimiter (except where required because it's a json string).

Consider using backtick quoted golang strings so you don't need to escape `"` in json strings either.